ember/app/ui/components/identity-addresses/component.js

```diff
@@ -21,24 +22,39 @@ export default class IdentityAddressesComponent extends Component {
   @service notification;
   @service store;
   @service intl;
-  // @service can;
 
   // Lifecycle
 
-  @action onUpdate() {
+  @action
+  onUpdate() {
+    this.fetchCountries.perform();
     this.fetchAddresses.perform(this.args.identity);
   }
 
   // List
 
   @lastValue("fetchAddresses") addresses;
-
-  @restartableTask *fetchAddresses(identity) {
+  @restartableTask
+  *fetchAddresses(identity) {
     return yield this.store.query("address", {
       filter: { identity: identity.id },
     });
   }
 
+  @lastValue("fetchCountries") countries;
+  @restartableTask
+  *fetchCountries() {
+    const adapter = this.store.adapterFor("identity");
+    adapter.headers["Accept-Language"] = localStorage.getItem("locale") ?? "en";
+    const response = yield fetch(adapter.buildURL("address"), {
+      method: "OPTIONS",
+      headers: adapter.headers,
+    });
+    const data = yield response.json();
+
+    return data.data.actions.POST.country.choices;
```

Comment:
Test for response.ok, only return choices if ok, else empty list

Reply:
Done